package.json

```diff
@@ -18,8 +18,8 @@
     "cypress:backend": "cd backend && yarn run dev",
     "cypress:webapp": "cd webapp && yarn run dev",
     "cypress:setup": "run-p cypress:backend cypress:webapp",
-    "cypress:run": "cross-env cypress run",
-    "cypress:open": "cross-env cypress open",
+    "cypress:run": "cross-env cypress run --browser firefox",
+    "cypress:open": "cross-env cypress open --browser firefox",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-20T22:17:38Z" title="Thursday, February 20th 2020, 11:17:38 pm +01:00">Feb 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -18,8 +18,8 @@
     "cypress:backend": "cd backend && yarn run dev",
     "cypress:webapp": "cd webapp && yarn run dev",
     "cypress:setup": "run-p cypress:backend cypress:webapp",
-    "cypress:run": "cross-env cypress run",
-    "cypress:open": "cross-env cypress open",
+    "cypress:run": "cross-env cypress run --browser firefox",
+    "cypress:open": "cross-env cypress open --browser firefox",
```


That has the drawback that we cannot record our cypress tests anymore :cry: can we close?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T22:50:25Z" title="Thursday, February 20th 2020, 11:50:25 pm +01:00">Feb 20, 2020</time>_

---


No, actually it doesn't, if you look at the build, or click on cypress, you can see that all the videos were recorded

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T22:53:06Z" title="Thursday, February 20th 2020, 11:53:06 pm +01:00">Feb 20, 2020</time>_

---


I wouldn't close, is there any justifiable reason to use a different browser than what we recommend our users to use for e2e tests?